WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -735,7 +737,8 @@ static GSourceFuncs _handlerIntervention =
             Core::JSON::DecUInt16 LocalStorageSize;
             Core::JSON::Boolean IndexedDBEnabled;
             Core::JSON::String IndexedDBPath;
-            Core::JSON::DecUInt16 IndexedDBSize; // [KB]
+            Core::JSON::Double IndexedDBSize; // [percentage of volume space for each domain]
```

Comment:
I'm not sure if we need it to be `Double`, depends whether we want to be able to be this precise or not, what do you think @modeveci?

It's a good point with the other remark as well, but I'm not sure if this is indeed a problem for backwards compatibility - right now if the WebKit API version is lower than 2.42, we cast the config value to `uint64_t` before pasting it to the WebKit API.

But if we decided that we don't need such level of precision, then we could just as easily solve it by casting to double for the WebKit API versions 2.42 or higher, like you said.

Reply:
Precise is not important. So we can keep as it is before and cast double during calling webkit apis.

Reply:
@modeveci @MFransen69 I changed `IndexedDBSize` and `IndexedDBTotalSize` to `DecUInt16`, and added casting accordingly. If we are keeping it exactly as before for backwards compatibility, I brought back multiplying by 1024 for any versions before 2.42, so that the configs with `IndexedDBSize` in KB won't be affected by this change 😄